spec/models/event_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Event, type: :model do
+  FactoryGirl.define do
+    factory :event do
+      title Faker::String.random(6..20)
+      place_title Faker::String.random(1..30)
+      place_cord Faker::String.random
+      description Faker::String.random(30..100)
+      start_at Time.now
+      end_at 1.week.from_now
+    end
+  end
+
+  %i[title place_title place_cord description start_at end_at].each do |attribute|
```

Comment:
Personally, I don't like specs generated in `each`.
Let's group validations by `attribute`.
Example:
```ruby
describe 'Validations' do
   describe '#title' do
     it { should validate_presence_of(:title) }
     
     it { should validate_length_of(:title).is_at_least(6).is_at_most(20) }
   end

   describe '#place_title' do
     # ..
   end
end

```